### PR TITLE
Merge/release 17.7 beta 3 into develop

### PR DESCRIPTION
**Pull request by mokagio**
Includes:

- A fix for blogging reminders in the Jetpack app (#16790) by @Gio2018 
- Fixes in the likes list and likes train (#16791) by @aerych 
- Add a frequency label to the day picker for blogging reminders (#16799) by @diegoreymendez 
- Refresh blogging reminder configurations in Site Settings (#16800) by @diegoreymendez 

I have a question on a conflict resolution, see inline comments for more details. If it's okay, then we can merge as long as CI is green.

### CODE REVIEW

File: WordPress/WordPress.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -23709,7 +23709,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.automattic.jetpack;
 				PRODUCT_MODULE_NAME = WordPress;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "WPiOS Development Profile";
+				PROVISIONING_PROFILE_SPECIFIER = "Jetpack iOS Development";
```
Comment (mokagio) [inline WordPress/WordPress.xcodeproj/project.pbxproj:23712]: This one may stand out as out of place, but it's part of #16790. What it does is setting the Jetpack app to use its development provisioning profile, which is appropriate.

I'm not sure what to make of the `PRODUCT_MODULE_NAME = WordPress;` in the Jetpack target but that is out of scope for this PR...

File: fastlane/jetpack_metadata/ar-SA/release_notes.txt [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-
-لم تتمكن أبدًا من تحرير صفحات أرشيف المقالات، نظرًا إلى أنه يتم إنشاؤها تلقائيًا. في الإصدار 17.6، لن تتمكن من فتحها في المحرِّر، ما يوفِّر عليك رحلة من دون فائدة.
-
-لقد عملنا كذلك على إصلاح أشياء:
-
-لقد اعتنينا ببعض الإشعارات التي كانت تتداخل مع ورقة الإجراءات، بما في ذلك الإشعارات التي تعرض عناصر التحكم في أيقونة الموقع، والأخرى التي تعرض إزالة البدء السريع، والثالثة التي تعرض المزيد من الإجراءات في المحرِّر.
-
-وأخيرًا، تخلصنا من خطأ كان يتسبب أحيانًا في تعطل التطبيق في أثناء إدارة التعليقات، وخطأ آخر لإصلاح خطأ تسجيل الدخول إلى مواقع WordPress.org لإظهار المُضمَّن.
-
-هذا كل ما يخص الإصدار 17.6، نراك في الإصدار 17.7!
```
Comment (mokagio) [inline fastlane/jetpack_metadata/ar-SA/release_notes.txt:15]: Not surprising to see this and the other release notes deleted. They were manually added as part of the 17.6 finalization and didn't get removed during code freeze or beta 1 because the automation from #16764 hadn't been merged yet.

File: fastlane/jetpack_metadata/he/description.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+להשתמש בכלים עוצמתיים לאבטחה ולביצועים ישירות מהטלפון עם Jetpack ל-iOS.
+
+לשחזר את האתר מכל מקום אם משהו משתבש. לסרוק את האתר לאיתור איומים ולפתור אותם בהקשה אחת. לעקוב אחר הפעילות באתר כדי לראות מי שינה מה ומתי. לבדוק את הנתונים הסטטיסטיים שלך כדי לראות מאיזו מדינה נכנסו היום המבקרים באתר שלך.
+
+האפליקציה תומכת גם באפשרויות המוכרות של WordPress. לשבת על הספה ולכתוב חמשיר. לצלם תמונה בהפסקת הצהריים ולפרסם אותה. לענות על התגובות האחרונות שקיבלת.
+
+האפליקציה של Jetpack ל-iOS תומכת באתרי WordPress.com שבהם מופעל השירות של Jetpack ושאליהם מחובר חשבון WordPress.com.
+
+ניתן לעיין בהצהרת הפרטיות למשתמשים מקליפורניה בכתובת https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.
```
Comment (mokagio) [inline fastlane/jetpack_metadata/he/description.txt:9]: This is sort of the opposite of the comment above regarding deleted release notes.

#16764 added back the Hebrew locale, resulting in all the already translated metadata (notice there's no 17.7 release notes) being downloaded this time around.

File: WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift [hunk 4]
```diff
@@ -335,8 +335,8 @@ extension SiteSettingsViewController {
             return ""
         }
 
-        let formatter = BloggingRemindersScheduleFormatter(schedule: scheduler.schedule(for: blog))
-        return formatter.shortIntervalDescription.string
+        let formatter = BloggingRemindersScheduleFormatter()
+        return formatter.shortIntervalDescription(for: scheduler.schedule(for: blog)).string
```
Comment (mokagio) [inline WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift:339]: @diegoreymendez I think this is the appropriate conflict resolution –it's the only one that compiles 😅 and it matches your changes from #16799– but could you confirm?

I'm asking because I was surprised by having a conflict here. Following the Git history, I can see #16781 went into `develop` recently (i.e.: after the code freeze) and that it does similar work to #16800. 

My guess is that #16800 is a refinement on the #16781 work, so this is the appropriate resolution. Let me know otherwise. Cheers!

File: WordPress/WordPress.xcodeproj/project.pbxproj (hunk 1 above)
Comment (AliSoftware) [reply WordPress/WordPress.xcodeproj/project.pbxproj:23712]: I think it's ok to keep the module name as `WordPress`, however odd it might look, because that's for use in symbol namespacing in the code, and if some code in the codebase uses `WordPress.SomeClass.someStaticSymbol` for example to disambiguate, we wouldn't want that code to fail when it's build by the Jetpack target just because of this.

Not sure we have any such disambiguation usage in the code tho, and if not, it could be acceptable and nice for consistency to change it to `Jetpack` for the jetpack target 🤷  (I _do_ like consistency…)

**Review by AliSoftware — COMMENTED** [review]
Looks good from the commit list and git branch matching what the description says it does, and from the content of tooling and localization files. 👍 

I'm not marking the PR as approved yet though because I want to make sure @diegoreymendez confirms the conflict resolution (https://github.com/wordpress-mobile/WordPress-iOS/pull/16807#discussion_r663649664) first, so I'll let him approve the PR, but the rest is ok with me.

File: WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift (hunk 4 above)
Comment (diegoreymendez) [reply WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift:339]: This looks correct to me!

File: WordPress/WordPress.xcodeproj/project.pbxproj (hunk 1 above)
Comment (diegoreymendez) [reply WordPress/WordPress.xcodeproj/project.pbxproj:23712]: I think this change may have been necessary for testing - honestly not sure.  The development profile being specific to Jetpack is in line with all our apps and libs, though, so I think this is fine.

Regarding the module name, I'd avoid making the change here if we don't need to.

**Review by diegoreymendez — APPROVED** [review]
Works fine, looks fine.
